src/node_sqlite.cc

```diff
@@ -1645,7 +1656,7 @@ void StatementSync::IterateReturnCallback(
 }
 
 void StatementSync::IterateNextCallback(
-    const FunctionCallbackInfo<Value>& args) {
+    const v8::FunctionCallbackInfo<v8::Value>& args) {
```

Comment:
We can remove the `v8::` once they are included with `using` already.

```suggestion
    const FunctionCallbackInfo<Value>& args) {
```